pages/src/navigation/sidebar/sidebar.ts

```diff
@@ -37,9 +37,13 @@ export class ESLDemoSidebar extends ESLToggleable {
     ESLMediaQuery.for('@+MD').removeListener(this.onBreakpointChange);
   }
 
+  protected storeState() {
+    this.open ? localStorage.removeItem('sidebar-collapsed') : localStorage.setItem('sidebar-collapsed', 'true');
```

Comment:
I guess it is enough to store any string except an empty string. You don't have to write 'true', you can just write '1'.